types/react-sortable-tree/react-sortable-tree-tests.tsx

```diff
@@ -68,9 +68,12 @@ class Test extends React.Component {
                     searchFinishCallback={(matches: NodeData[]) => {
                         const firstTitle = matches[0].node.title;
                     }}
-                    generateNodeProps={(data: ExtendedNodeData) => ({
-                        buttons: [data.node.title]
-                    })}
+                    generateNodeProps={(data: ExtendedNodeData) => {
+                        const { parentNode } = data;
```

Comment:
Do we need the `parentNode` if it's not used?

Reply:
Didn't know how better update tests 🤷‍♂️

Reply:
It would be OK without that line. We don't use other stuff either.

Reply:
ok